app/src/test/java/org/mozilla/focus/utils/AnyNonNull.kt

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.utils
+
+import org.mockito.Mockito
+
+/**
+ * A normal [Mockito.any] is a nullable type, so this allows us to test nonnull
```

Comment:
You should mention that the reason this is required is because it crashes otherwise, i.e. "use this when `any()` crashes".